perf/benchmark/configs/istio/security_authz/latency.yaml

```diff
@@ -0,0 +1,18 @@
+# Data: latency
+# Config: security_authz
+telemetry_mode: "none"
```

Comment:
if you did not change telemetry setting, this mode should be "v2-stats-nullvm"

Reply:
update this and remove the do-not-merge label

Reply:
changed